vscode/extension/.vscode/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+#Change Log<br/>
+
+All notable changes to the "ballerina-integrator" extension will be documented in this file.
+
+### [Unreleased]
+Initial Release
+
```

Comment:
Shall we change this to the original format? 
https://github.com/wso2/ballerina-integrator/pull/95/files#diff-a3cdb1ec6e4e0a13382c3c41b0068230

Reply:
Fixed